src/SFA.DAS.Payments.Monitoring.Jobs.Client/JobMessageClient.cs

```diff
@@ -19,71 +21,102 @@ public class JobMessageClient : IJobMessageClient
     {
         private readonly IMessageSession messageSession;
         private readonly IPaymentLogger logger;
+        private readonly IConfigurationHelper config;
 
-        public JobMessageClient(IMessageSession messageSession, IPaymentLogger logger)
+        public JobMessageClient(IMessageSession messageSession, IPaymentLogger logger, IConfigurationHelper config)
         {
             this.messageSession = messageSession ?? throw new ArgumentNullException(nameof(messageSession));
             this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
+            this.config = config ?? throw new ArgumentNullException(nameof(config));
         }
 
-
         public async Task ProcessedJobMessage(long jobId, Guid messageId, string messageName, List<GeneratedMessage> generatedMessages)
         {
-            //logger.LogVerbose($"Sending request to record successful processing of event. Job Id: {jobId}, Event: id: {messageId} ");
-            //var itemProcessedEvent = new RecordJobMessageProcessingStatus
-            //{
-            //    JobId = jobId,
-            //    Id = messageId,
-            //    MessageName = messageName,
-            //    EndTime = DateTimeOffset.UtcNow,
-            //    GeneratedMessages = generatedMessages ?? new List<GeneratedMessage>(),
-            //    Succeeded = true
-            //};
-            //await messageSession.Send(itemProcessedEvent).ConfigureAwait(false);
-            //logger.LogDebug($"Sent request to record successful processing of event. Job Id: {jobId}, Event: id: {messageId} ");
+            try
+            {
+                logger.LogVerbose($".NET 6 Nuget - Sending request to record successful processing of event. Job Id: {jobId}, Event: id: {messageId} ");
+               
+                var batchSize = 1000; //TODO: this should come from config
```

Comment:
Are we ok to leave this for new and raise a tech debt ticket to address later?

Reply:
Probably one for @abroodie. I suppose we should see if the need occurs for it?